SDK/simVis/DisableDepthOnAlpha.cpp

```diff
@@ -57,7 +57,15 @@ void DisableDepthOnAlpha::setValues(osg::StateSet* stateset, int value)
 {
   if (stateset == NULL)
     return;
-  stateset->setDefine(USE_DISABLE_DEPTH_DEFINE, value);
+
+  int over = value & osg::StateAttribute::OVERRIDE;
+  int prot = value & osg::StateAttribute::PROTECTED;
+
+  int negValue = (value | osg::StateAttribute::ON)?
```

Comment:
This test will always pass.  value | ON is always non-zero

Do you mean `value & osg::StateAttribute::ON` ?

Reply:
Indeed I did. Thanks